extensions/wikia/Search/classes/Result.php

```diff
@@ -312,16 +381,20 @@ public function offsetGet( $key ) {
 	 * Replace unusual URL escape codes with their equivalent characters
 	 *
 	 * @param $url String
+	 *
 	 * @return String
 	 *
 	 * @todo  This can merge genuinely required bits in the path or query string,
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this "TODO" comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1135)